Makefile

```diff
@@ -51,6 +56,11 @@ clean-test:
 	-for image in $$(docker images | awk '/teradatalabs\/pa_test/ {print $$1}'); do docker rmi -f $$image ; done
 	@echo "\n\tYou can kill running containers that caused errors removing images by running \`make clean-test-containers'\n"
 
+clean-docs:
```

Comment:
clean should probably also depend on clean-docs
clean-docs should be .PHONY. See line 1. Actually, all of the clean targets should be .PHONY. Could you please add them while you're fixing this?